### DISCUSSION TITLE
How about extending with the private API for integration purposes and automated order execution

### DISCUSSION THREAD

**Discussion by RadoslawMerkx** [HansvBug/Bitvavo-Coin-Monitor#4] [Ideas]
If the private API is used then you have the ticker info and besides can manage orders. Having these together is a great basis for creating order tooling, such as bots, notifiers, portfolio management. What do you think about that? Could be a joint effort.

**Comment by HansvBug** [comment]
Hi,

I haven't used c# in about 2 years. So I don't think I can easily expand
it. In addition, I would set it up differently now. I would set it up with
an MVVM structure I think. And I've never done that with c#. If you want to
modify it (necessary to get it working again) then you can download it
yourself and continue. I'm currently too busy setting up another tool in an
other language. Perhaps i will look at it later.

What country are you from?

Greatings

Op zo 7 jul 2024 om 10:44 schreef RadoslawMerkx ***@***.***>:
